### ISSUE TITLE
Cannot add helm repository

### ISSUE THREAD

**Issue by mojochao** [aslafy-z/helm-git#79]
I am trying to add a helm repository using helm-git plugin as follows:

    $ helm repo add traefik-helm-chart "git+https://github.com/containous/traefik-helm-chart@?ref=master"
    Error: unknown command "home" for "helm"
    Run 'helm --help' for usage.
    Error: looks like "git+https://github.com/containous/traefik-helm-chart@?ref=master" is not a valid chart repository or cannot be reached: plugin "helm-git" exited with error

I don't see any `helm home` command, so I'm assuming this is coming from the plugin.  Since this used to work, did `helm` recently remove a `home` subcommand?   Is there any way to get debug output from the plugin itself?

Output of `helm version`:

    $ helm version
    version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}

    $ helm plugin list
    NAME    	VERSION	DESCRIPTION
    diff    	3.1.1  	Preview helm upgrade changes as a diff
    helm-git	0.5.0  	Get non-packaged Charts directly from Git.
    registry	0.7.0  	This plugin provides app-registry client to Helm.
        	       	usage:
        	       	  $ helm reg...

Output of `kubectl version`:

    $ kubectl version
    Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.4", GitCommit:"8d8aa39598534325ad77120c120a22b3a990b5ea", GitTreeState:"clean", BuildDate:"2020-03-12T23:41:24Z", GoVersion:"go1.14", Compiler:"gc", Platform:"darwin/amd64"}
    Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.9-eks-502bfb", GitCommit:"502bfb383169b124d87848f89e17a04b9fc1f6f0", GitTreeState:"clean", BuildDate:"2020-02-07T01:31:02Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 

AWS EKS 1.14

**Comment by mojochao** [comment]
I hoped this was fixed by https://github.com/aslafy-z/helm-git/commit/e3ce73196944de5fd15eb36fcf630729a0cd5ada, however when I remove the `0.5.0` version and add again with:

    $ helm plugin install https://github.com/aslafy-z/helm-git.git

I still see the error:

    $ helm repo add traefik-helm-chart "git+https://github.com/containous/traefik-helm-chart@?ref=master"
    Error: unknown command "home" for "helm"
    Run 'helm --help' for usage.
    Error: looks like "git+https://github.com/containous/traefik-helm-chart@?ref=master" is not a valid chart repository or cannot be reached: plugin "helm-git" exited with error

Any ideas on what's still going on?  Did I not pick up the correct changes with the plugin?

**Comment by mojochao** [comment]
I cannot seem to install plugin with master branch.  It says it is installed, but the `helm plugin list` command still shows v0.5.0.  If I examine the contents of `~/Library/helm/plugins/helm-git.git/` directory it does not reflect the HEAD ref of master branch.

I've copied the contents of the HEAD ref of master branch into my installed plugins directory, and it works.

When might another release incorporating this fix be expected?  Alternatively, is there a way to really install plugin with HEAD ref of master branch?

Many thanks!

**Comment by aslafy-z** [comment]
I just released [`v0.6.0`](https://github.com/aslafy-z/helm-git/releases/tag/v0.6.0).

You can install it by running
```
helm plugin install https://github.com/aslafy-z/helm-git.git --version 0.6.0
```

Let me know if it fixes your problem!

**Comment by mojochao** [comment]
That fixed it.  Many thanks!  Note that I had to change the install command from:
```
helm plugin install https://github.com/aslafy-z/helm-git.git --version 0.6.0
```
to:
```
helm plugin install https://github.com/aslafy-z/helm-git --version 0.6.0
```
Use of the trailing `.git` in the URL is also documented in the README.

**Comment by aslafy-z** [comment]
Fixed with 688a64e, thank you!
